consumers/notifiers/service_test.go

```diff
@@ -29,7 +29,8 @@ func newService() notifiers.Service {
 	auth := mocks.NewAuth(map[string]string{exampleUser1: exampleUser1, exampleUser2: exampleUser2, invalidUser: invalidUser})
 	notifier := mocks.NewNotifier()
 	idp := uuid.NewMock()
-	return notifiers.New(auth, repo, idp, notifier)
+	from := ""
```

Comment:
Mayeb better to don't use an empty string here?

Reply:
I agree, simply use a string literal.